Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/SrgSemantics.azsli

```diff
@@ -59,6 +59,7 @@ ShaderResourceGroupSemantic SRG_Bindless
 ShaderResourceGroupSemantic SRG_RayTracingGlobal
 {
     FrequencyId = 0;
+    ShaderVariantFallback = 128; // The minimal size to use Shader Options.
```

Comment:
Are we introducing shader options to a shader using this srg?

Reply:
In some cases AZSLc will generate an error if ShaderVariantFallback isn't declared, so I added it to avoid this condition.